veps/sig-network/hotpluggable-nad-ref.md

```diff
@@ -0,0 +1,184 @@
+# VEP #140: Update Interface Network Attachment without restart
+
+## Release Signoff Checklist
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [ ] (R) Enhancement issue created, which links to VEP dir in [kubevirt/enhancements] (not the initial VEP PR)
+- [ ] (R) Target version is explicitly mentioned and approved
+- [ ] (R) Graduation criteria filled
+
+## Overview
+
+Currently, the association between a VM Interface and a Network Attachment Definition (NAD) is immutable for the lifecycle of a VirtualMachineInstance (VMI).
```

Comment:
VMs using secondary networks, refer to a NetworkAttachmentDefinition object's namespace and/or name as part of their network specification. Currently, the whole network specification, including this reference - is immutable.